ABWL-LEDER/appendix/LKF.md

```diff
@@ -559,105 +683,260 @@ einfache Programmierbarkeit und Aufnahme in die EDV bei Rechnersystemen | Kennza
 
 ### Sie kennen die zwei Abschnitte der Projektumsetzung!
 
-- TODO
+- Projektfindung/-vorbereitung
+- Projektrealisierung
 
 ### Sie können den PDCA als Führungsmethodik am konkreten Beispiel anwenden!
 
-- TODO
+- Plan
+- Do
+- Check
+- Act
 
 ### Sie kennen die vier Abschnitte des Gesamtprozesses der Projektumsetzung!
 
-- TODO
+- Prozessvorbereitung
+  - Problemermittlung
+  - Problemanalyse
+  - Alternativen
+  - Erfolgseinschätzung
+  - Entscheidung
+- Projektplanung
+  - Aufgabenplanung
+  - Personalplanung
+  - Terminplanung
+  - Ergänzungen
+  - Planungsergebnisse
+- Projektgestaltung
+  - Projektauslösung
+  - Projektarbeiten
+  - Projektsteuerung
+- Projekteinführung
+  - Lösungseinführung
+  - Abschlussarbeiten
 
 ### Sie können zwei Methoden zum Finden von Fehlern/Problemen erläutern!
 
-- TODO
+- Ishikawa (Ursache-Wirkungs-Diagramm) (7M!)
+- FMEA (Formblatt mit Punktesystem)
 
 ### Sie kennen mind. drei Kreativtechniken mit denen man Ideen entwickeln kann!
 
-- TODO
+- Brainstorming
+- Brainwriting (mit 635 Methode)
+- Morphologischer Kasten (Konfigurator)
+- Mindmapping (graphische Darstellung von Sachverhalten)
+- Bionic (Inspiration für die Problemlösung in der Natur suchen)
 
 ### Sie können die Risiken bei der Umsetzung an einem Ampelsystem erläutern und die Bedeutung und den Inhalt der Nachhaltigkeit darstellen!
 
-- TODO
+- TODO: **Jonathan**: PM II.pdf Folie 34, Qualitätsmanagement.pdf Seite 76 noch mal
```

Comment:
Inhalt der Nachhaltigkeit = Drei-Säulenmodell der Nachhaltigkeit?

![image](https://user-images.githubusercontent.com/21085384/142653654-41784006-b97e-4274-9233-4c430521f283.png)

```suggestion
- TODO: **Jonathan**: PM II.pdf Folie 34, Qualitätsmanagement.pdf Seite 76 noch mal

$$\begin{matrix}
\text{Soziale Bez.}&&\text{Ökonomie}\\
\qquad\searrow&&\swarrow\qquad\\
&\textbf{Nachhaltigkeit}\\
&\uparrow\\
&\text{Ökologie}\end{matrix}$$
```

Reply:
Boah, diese Arts des Zeichnens in LaTeX muss ich mir mal merken.

Reply:
Also ja, ich glaub diese drei Sachen sind Bestandteile der Nachhaltigkeit ... aber was haben die hier in dem Kontext mit dem Ampelsystem zu tun 🤔

Reply:
> Also ja, ich glaub diese drei Sachen sind Bestandteile der Nachhaltigkeit ... aber was haben die hier in dem Kontext mit dem Ampelsystem zu tun 🤔

zweite Teil der Frage hat mit dem ersten nichts zu tun.
Amplesystem: nicht ERP, sondern Risikobewertung?
Wir sollten Fragen auftrennen

Reply:
Hab die Fragen mal getrennt, Yannis' Lösung übernommen und mal kurz den verwiesenen Inhalt der Folie in der Frage davor dargestellt. Schaut euch mal an, ob das so passt.